src/js/components/Page/index.d.ts

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { BoxProps } from '../Box/index';
+
+export interface PageProps {
+  contentProps?: BoxProps;
```

Comment:
What's a scenario where we would need `contentProps`? I'm wondering if this is infrequent enough that callers should just use their own PageContent.

Reply:
Yeah that's reasonable, I'll remove contentProps